WhoReacted/WhoReacted.plugin.js

```diff
@@ -156,34 +159,48 @@ module.exports = !global.ZeresPluginLibrary ? class {
             const VoiceUserSummaryItemComponent = WebpackModules.find(m => m.default?.displayName === "VoiceUserSummaryItem").default;
 
             const Reaction = await this.findReaction();
-            const Reactions = await this.findReactions();
+            const Reactions = this.findReactions();
```

Comment:
I think it's better to find the Reactions using the ReactComponent util to reuse again below when forcing update.